### ISSUE TITLE
expr function should work like where function

### ISSUE THREAD

**Issue by sergiofilhowz** [hiddentao/squel#147]
Using where it's possible to pass more than one argument, example:

    squel.select().from('table').where('field = ? and field = ?', 1, 2);

Using expr it's not possible

    squel.expr().and('field = ? and field = ?', 1, 2);

![captura de tela 2015-04-29 as 19 39 13](https://cloud.githubusercontent.com/assets/4141723/7402975/7461b642-eea7-11e4-8e65-2a1768d6cf43.png)

**Comment by Smert** [comment]
Expression separately determines conditional constructions (AND, OR). You can use two AND.

**Comment by sergiofilhowz** [comment]
I would like to use with BETWEEN, unfortunately its not possible with expr

**Comment by Smert** [comment]
```
squel.expr()
  .and('date >= ?', dateMin)
  .and('date <= ?', dateMax);
```
So it looks right.

**Comment by hiddentao** [comment]
Yep, `Expression` is in need of a serious upgrade at some point to support all of these use-cases.
